source/2019-07-05-the-ember-times-issue-105.md

```diff
@@ -39,12 +40,13 @@ Hoi Emberistas! 🐹
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [To Sync Or Not to Sync: Observers 2.0 ⏱](https://emberjs.github.io/rfcs/0494-async-observers.html)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In an effort to remove legacy code from Ember, promote better programming practices in apps and lay the foundation for clearer code paths, a [recently accepted RFC (Request for Comments)](https://github.com/emberjs/rfcs/pull/494) proposes an important change to the **observer** APIs: the [`addObserver`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fobservers/addObserver), [`removeObserver`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject%2Fobservers/removeObserver) and the [`observer`](http://api.emberjs.com/ember/3.10/functions/@ember%2Fobject/observer) functions shall be configurable to operate either synchronously (the way observers always used to work in Ember apps since the pre-v1.0 days) or asynchronously.
```

Comment:
This question is more for my understanding: Is it better to direct readers to a specific version (3.10) of api.emberjs.com instead of release?

Reply:
I read the Summary in https://emberjs.github.io/rfcs/0494-async-observers.html, and thought brief definitions of **synchronous** and **asynchronous** could help (defining asynchronous in the context of observer, in particular). What do you think?

synchronous = observers are called immediately after the property they are observing has changed
asynchronous = observers are called during the next runloop

Reply:
> This question is more for my understanding: Is it better to direct readers to a specific version (3.10) of api.emberjs.com instead of release?

I think it's in fact better to link to the `releases` version of the docs and just updated that accordingly :+1:

> I read the Summary in https://emberjs.github.io/rfcs/0494-async-observers.html, and thought brief definitions of synchronous and asynchronous could help (defining asynchronous in the context of observer, in particular). What do you think?

Yes, that sounds great - I added two more short sentences to highlight that and thank you for the suggestion :sparkles: